spec/order_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'order'
+
+describe Order do
+  describe "#initialize" do
+    # I would normally think this test as unecessary, but in this case I want to make sure that
+    # if the input method for the menu changes, it muct still return an array. Is that correct?
+    let(:menu) {[]}
+    it "should initialize with a menu array" do      
+      expect(subject.menu).to be_a Array 
+    end
+  end 
+
+  describe "#add_to_order" do
+    # is this test insane?
+    it "should add the quantity ordered to the correct dish" do
+      menu = [{:dish => "Burgers", :price => 10}]

```

Comment:
The test is definitely sane, but ideally the instance of `Total` should be mocked/stubbed and injected into `Order` via the `.new` method — right now, there's a `Total.new` instantiated at line 20, but it's never used, since the instance used is the default one (the default argument from the `initialize` definition, because `Order.new` is not being called with it)